### PR TITLE
Add / remove some files from backup

### PR DESCRIPTION
**Pull request by vuntz**
It's obviously needed by crowbar.

### CODE REVIEW

**Comment by nkrinner** [comment]
+1

File: backup/crowbar-backup [hunk 1]
```diff
@@ -59,14 +59,14 @@ function echo_summary ()
...
-CROWBARFILES="etc/crowbar var/lib/crowbar opt/dell/crowbar_framework/config/client.pem"
+CROWBARFILES="etc/crowbar etc/crowbar.install.key var/lib/crowbar opt/dell/crowbar_framework/config/client.pem"
 TFTPFILES="srv/tftpboot/validation.pem srv/tftpboot/nodes"
 ROOTFILES="root/.chef root/.ssh root/.gnupg"
 BFILES="$ROOTFILES $CHEFFILES $CROWBARFILES $TFTPFILES etc/resolv.conf.forwarders"
 
 # files and directories to be excluded from backup
 EXCLUDEFILES="var/lib/crowbar/cache"
-XFILES=""
+XFILES="etc/chef/amqp_passwd"
```
Comment (bmwiedemann) [inline backup/crowbar-backup]: should go one line above to get the --exclude prefix below

Comment (vuntz) [inline backup/crowbar-backup]: fixed

**Comment by bmwiedemann** [comment]
+1

**Pull request merged**